frontend/src/components/SelectList/index.js

```diff
@@ -11,7 +11,7 @@ const SelectList = ({ name, options, value, multiple = false, onChange }) => {
   };
 
   return (
-    <div className="select-list">
+    <div className="select-list" name={name}>
```

Comment:
A best practice [recommended by cypress](https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements) is to use `data-cy` as the selector to identify elements in tests.

Reply:
Good point! I was trying to not be very invasive in the rendered html (yet), but I can add the  `data-cy` attribute